docs/dataloader.adoc

```diff
@@ -0,0 +1,59 @@
+= fastai.dataloader
+
+== Introduction and overview
+
+*Note:* the fastai DataLoader has a similar API to the PyTorch DataLoader. Please see http://pytorch.org/docs/master/data.html#torch.utils.data.DataLoader[the PyTorch documentation] for usage and details. The documentation presented here focuses on the differences between the two classes.
+
+== {{class DataLoader, dataset, batch_size=1, shuffle=False, sampler=None, batch_sampler=None, pad_idx=0, num_workers=None, pin_memory=False, drop_last=False, pre_pad=True, half=False, transpose=False, transpose_y=False}}
+
+.Used to iterate through a dataset to pass data into a model for training.
+
+=== {{arguments}}
+
+For information on arguments with no descriptions, please see http://pytorch.org/docs/master/data.html#torch.utils.data.DataLoader[the PyTorch documentation]
+
+dataset::
+
+batch_size::
+
+shuffle::
+
+sampler::
+
+batch_sampler::
+
+pad_idx (type int, default 0)::
+    A padding index representing how many zeros to add to each batch. See: `pre_pad`
+
+num_workers (type int, default `None`)::
+    Allows the user to manually set the number of workers. If left as `None`, it will default to the number of CPU cores the system has. If > 0, the dataloader will create `num_workers` number of jobs using `concurrent.futures.ThreadPoolExecutor`.
+
+pin_memory::
+
+drop_last::
+
+pre_pad (type bool, default `True`)::
+    If `pad_idx` is non-zero, this determines if the zeros should go at the beginning of the batch, or at the end. By default, the zeros are added at the beginning of the batch.
+
+half (type bool, default `False`)::
+    If `True`, `torch.cuda.HalfTensor()` will be used instead of `torch.FloatTensor()`.
+
+transpose (type bool, default `False`)::
+    If `True`, each batch will have its inputs transposed.
+
+transpose_y (type bool, default `False`)::
+    If `True`, each batch will have its outputs (labels) transposed.
+
+=== {{methods}}
+
+{{method jag_stack(b),}}
```

Comment:
These should be in the format: `{{method jag_stack,b}}`. i.e. the list of params comes after the comma. It's possible the template script isn't handling this correctly - in which case feel free to have a go at fixing it! :)